src/schema/suffixes/_template.yaml

```diff
@@ -0,0 +1,3 @@
+---
+name:
+description:
```

Comment:
is this template supposed to be committed?

Reply:
I like to share the template so it's easier to just copy over to a new file as needed. I don't think it causes any problems (at least it doesn't seem to have with the metadata files), but I can remove it if there's an issue.

Reply:
okay, that's fine - just wanted to avoid an accidental commit